### DISCUSSION TITLE
How to set a custom layout?

### DISCUSSION THREAD

**Discussion by iamajvillalobos** [excid3/revise_auth#92] [Q&A] [answered]
For example I want to use another layout for auth. I don't seem to read how to do that in the readme. Usually in the controller you can set the `layout 'auth'`.

**Comment by excid3** [comment, marked as answer]
Just override the controller?
```
# app/controllers/revise_auth_controller.rb
class ReviseAuthController < ::ApplicationController
  layout "auth"
end
```

**Reply by iamajvillalobos** [reply]
Thank you! I was thinking about that too last night, but maybe there were some configurations I was just not using.
